src/components/Header/Header.jsx

```diff
@@ -303,15 +309,15 @@ const Header = ({
                       <ListItemText primary={strings.app_my_profile} />
                     </ListItem>
                   </DrawerLink>
-                  <DrawerLink as="a" href={`${process.env.REACT_APP_API_HOST}/logout`}>
+                  <DrawerLink as="a" to={`${process.env.REACT_APP_API_HOST}/logout`}>
```

Comment:
I'm pretty sure this needs to be href since the logout is not part of the app

Reply:
It works well with the login button in local env.  Thought it will work here too.